tests/test_noema_orchestrator_workflow_contract.py

```
@@ -0,0 +1,119 @@
+    )
+    assert "mark_unconfigured()" not in workflow
+    assert "review skipped until Noema is deployed" not in workflow
+    assert "Noema app token is unavailable; review skipped." not in workflow
+    assert "COPILOT_GITHUB_TOKEN" not in workflow
+    assert "secrets: inherit" not in workflow
+
+
+def test_strix_nim_defaults_and_noema_sidecar_fail_closed(tmp_path: Path) -> None:
+    """Keep the Strix NIM empty-output contract; Noema now fails closed without the sidecar."""
+    strix_output = tmp_path / "strix-output"
+    strix = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)